rollup.config.js

```diff
@@ -36,6 +47,7 @@ const mainFile = [
 ];
 
 const generateConfig = async () => {
+  // eslint-disable-next-line compat/compat
```

Comment:
Disable eslint rules it not a good idea...maybe we should  change compatibility settings in the package json, or add some polyfills instead of disabling compatibility validations

Reply:
I totally agree! I blame it on eslint!